packages/field-label/src/FieldLabelMixin.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {
+    CSSResultArray,
+    html,
+    nothing,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import { property } from '@spectrum-web-components/base/src/decorators.js';
+import '@spectrum-web-components/icons-ui/icons/sp-icon-asterisk100.js';
+
+import styles from './field-label.css.js';
+import asteriskIconStyles from '@spectrum-web-components/icon/src/spectrum-icon-asterisk.css.js';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import { ObserveSlotText } from '@spectrum-web-components/shared';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T> = new (...args: any[]) => T;
+
+export declare class FieldLabelMixinInterface {
+    disabled: boolean;
+    required: boolean;
+    sideAligned: 'start' | 'end';
```

Comment:
I think this one should be optional:
```suggestion
    sideAligned?: 'start' | 'end';
```